packages/codemods/transforms/v3/radio-card/index.js

```diff
@@ -0,0 +1,63 @@
+const CODEMOD_ANALYSIS = process.env.CODEMOD_ANALYSIS;
+
+module.exports = function ({ source /*, path*/ }, { parse, visit }) {
```

Comment:
according to the changelog https://github.com/hashicorp/design-system/pull/1634/files#diff-fa1aa341f6153e2733520f2d8227380e02de23c96082cac195a30f4ddde9fbe7 also a `@maxWidth` should be applied. If this is the case, how is the codemod going to take care of it?

Reply:
Maybe something to add to the README for this codemod?

Reply:
That's a good shout – added an extra note to the `v3/radio-card` instructions